app/build.gradle

```diff
@@ -29,6 +38,7 @@ dependencies {
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     implementation 'com.telenor.connect:connect-android-sdk:1.7.1'
+    implementation project(':possumgather')
```

Comment:
Why don't we need `implementation project(':core')` here?

Reply:
I assume it is because possumgather has this in its build.gradle.

Reply:
Ok-dokey.